scripts/custom_lint_checks.py

```diff
@@ -520,13 +573,31 @@ def _compare_different_args(
                                 not_needed_type_in_docstring)
 
     def check_single_constructor_params(self, class_doc, init_doc, class_node):
+        """Check single constructor Parameters
```

Comment:
The explanation seems to be wrong -- this function checks whether the Class as well as the __inti__() method both have documentation. If they do, it adds an error message "multiple-constructor-doc".
Also, please always end a sentence with a period if its complete in meaning.